Dockerfile

```diff
@@ -1,33 +1,37 @@
-FROM tomcat:8-jre8
+FROM tomcat:9-jre8-alpine
```

Comment:
We tend to avoid alpine images to not use too many different OSes.
This won't reduce the image by much compared to the rest we install in there.

Reply:
Agreed

Reply:
In addition, alpine linux ships with a custom libc (musl), which might not react exactly as the classic glibc one.